src/App.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import Welcome from './components/calculator';
```

Comment:
- [ ] It will be very good and readable if this is imported as `Calculator` instead of ~`Welcome`~
```suggestion
import Calculator from './components/calculator';
```